scoring/src/main/scala/ai/h2o/sparkling/ml/models/H2OMOJOModel.scala

```diff
@@ -56,6 +58,22 @@ class H2OMOJOModel(override val uid: String) extends H2OMOJOModelBase[H2OMOJOMod
       withPredictionDf.drop(getDetailedPredictionCol())
     }
   }
+
+  protected override def applyPredictionUdfToFlatDataFrame(
+      flatDataFrame: DataFrame,
+      udfConstructor: Array[String] => UserDefinedFunction,
+      inputs: Array[String]): DataFrame = {
+    val relevantColumnNames = flatDataFrame.columns.intersect(inputs)
+    val args = relevantColumnNames.map(c => flatDataFrame(s"`$c`"))
+    val udf = udfConstructor(relevantColumnNames)
+    val predictWrapper = H2OMOJOCache.getMojoBackend(uid, getMojoData, this)
+    predictWrapper.getModelCategory match {
+      case ModelCategory.Binomial | ModelCategory.Regression | ModelCategory.Multinomial =>
+        flatDataFrame.withColumn(outputColumnName, udf(struct(args: _*), lit(0.0)))
```

Comment:
I think comment would help for `lit(0.0)`

Reply:
Ok, will add something.

Reply:
thank you!